src/System.Security.Cryptography.Csp/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netcoreapp-Windows_NT;
+      netstandard-Unix;
+      netcoreapp-Unix;
```

Comment:
if the test projects are exactly the same (meaning they don't have a condition in them includding more tests or defining constants) then netstandard is prefered. The reason is that if you have only the netstandard configuration, that would get built and tested in all verticals (netcoreapp, uap, uapaot, netfx). If you only have the netcoreapp configuration, it won't be tested in other platforms.